### ISSUE TITLE
New "date and time format" dialog

### ISSUE THREAD

**Issue by CocoCR300** [CocoCR300/flauncher#18]
The current one is difficult to use with a TV remote control.

**Comment by Pentaphon** [comment]
Yes! Can we have commonly-used presets? Typing in letters is annoying to begin with. 

https://devhints.io/datetime

**Comment by jdevrie** [comment]
I have to agree here with @CocoCR300 that the date and time format dialog is very hard to use on my Philips TV.
It has a lot of unexpected behavior like hangs and weird reactions to the remote or my keyboard.
